src/components/Connections/Connections.tsx

```diff
@@ -85,6 +85,8 @@ const truncateString = (input: string) => {
 };
 
 const headers = ['Connection details', 'Source → Destination', 'Last sync'];
+const isSortable = [true, false, false]; // connection details is a sortable column
```

Comment:
Can we combine headers and isSortable in a single object.

Maybe something like headers=[{ name: "Connection details", sortable: true }]

Reply:
@mdshamoon The reason I didn't do that is because, if we are creating a simple list using this component with no sorting, even then we would have to specify the sortable property in the header array. We would also have to update all the instances where the list component is currently used. So I thought I would add sorting options as seperate & optional props.
What do you think?

Reply:
Makes sense. I am still leaning towards adding the sortable option in the object in the array as it makes it more explicit and easy to understand which column is sortable. 

Let's do this for now combine both in a single object. 

```
const header =  {values: ['Connection details', 'Source → Destination', 'Last sync'],
sortable: [true, false, false]};
```
so that its easy to understand and map the sorted columns